static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -65,17 +65,13 @@ const ConsecutiveDBQueriesSpanEvidence = ({
   offendingSpans,
 }: SpanEvidenceKeyValueListProps) => (
   <PresortedKeyValueList
-    data={
-      [
-        makeTransactionNameRow(event),
-        causeSpans
-          ? makeRow(t('Starting Span'), getSpanEvidenceValue(causeSpans[0]))
-          : null,
-        ...offendingSpans.map(span =>
-          makeRow(t('Parallelizable Span'), getSpanEvidenceValue(span))
-        ),
-      ].filter(Boolean) as KeyValueListData
-    }
+    data={[
+      makeTransactionNameRow(event),
+      ...(causeSpans
+        ? [makeRow(t('Starting Span'), getSpanEvidenceValue(causeSpans[0]))]
+        : []),
+      makeRow('Parallelizable Spans', getMultiSpanEvidenceValues(offendingSpans)),
```

Comment:
```suggestion
      makeRow('Parallelizable Spans', spans.map(getSpanEvidenceValue)),
```

is even better IMO, `getMultiSpanEvidenceValues` isn't capturing a lot of value

Reply:
Updated!